external_search.py

```diff
@@ -268,375 +276,59 @@ def base_index_name(self, index_or_alias):
 
         return base_works_index
 
-    def query_works(self, library, query_string, media, languages, fiction, audiences,
-                    target_age, in_any_of_these_genres=[], on_any_of_these_lists=None, fields=None, size=30, offset=0):
```

Comment:
All this filtering information (in_any_of_these_genres, etc.) now lives in the Filter object.